### ISSUE TITLE
Feature Request: Find/Replace

### ISSUE THREAD

**Issue by TimothyHD** [tomboy-notes/tomboy-ng#212]
It would be nice to have a find/replace option

**Comment by davidbannon** [comment]
Yes, I am not sure why I don't do find/replace.  Should not be too hard.  I need to do some work there anyway, the existing find is not, IMHO, satisfactory anyway, from memory it has a switch to reverse search direction that does not work.

I will take 'Replace' on notice !

Thanks for the suggestion.
